src/components/forms/validations.jsx

```diff
@@ -44,5 +45,5 @@ module.exports.validationHOCFactory = function (defaultValidationErrors) {
 };
 
 module.exports.defaultValidationHOC = module.exports.validationHOCFactory({
-    isDefaultRequiredValue: 'This field is required'
+    isDefaultRequiredValue: <intl.FormattedMessage id="teacherRegistration.validationRequired" />
```

Comment:
I think we should move this string it to general if it's at the component level.